### PR TITLE
Reorder 2/os/__init__.py to look like 3/os/__init__.py

### PR DESCRIPTION
**Pull request by euresti**
This was a simple reordering of the lines in the file so that it looks like 3/os/__init__.py.
This should let us more easily combine the files and move them to 2and3

### CODE REVIEW

File: stdlib/2/os/__init__.pyi [hunk 1]
```diff
@@ -8,16 +8,95 @@ from . import path
...
+O_EXCL = 0
+O_TRUNC = 0
+O_DSYNC = 0    # Unix only
+O_RSYNC = 0    # Unix only
+O_SYNC = 0     # Unix only
+O_NDELAY = 0   # Unix only
+O_NONBLOCK = 0  # Unix only
+O_NOCTTY = 0   # Unix only
+O_SHLOCK = 0   # Unix only
+O_EXLOCK = 0   # Unix only
```
Comment (ambv) [inline stdlib/2/os/__init__.pyi]: O_SHLOCK and O_EXLOCK don't appear to be there for me on Linux. They are there on macOS though.

File: stdlib/2/os/__init__.pyi [hunk 2]
```diff
@@ -8,16 +8,95 @@ from . import path
...
+O_BINARY = 0     # Windows only
+O_NOINHERIT = 0  # Windows only
+O_SHORT_LIVED = 0  # Windows only
+O_TEMPORARY = 0  # Windows only
+O_RANDOM = 0     # Windows only
+O_SEQUENTIAL = 0  # Windows only
+O_TEXT = 0       # Windows only
```
Comment (ambv) [inline stdlib/2/os/__init__.pyi]: We should probably use the `sys.platform` check for those.

Comment (euresti) [inline stdlib/2/os/__init__.pyi]: I agree.  But this change is mostly a re-ordering. I can do that in a future diff?  This is mostly a reordering to make the files mostly identical and easier to diff/modify.

Comment (ambv) [inline stdlib/2/os/__init__.pyi]: Yes, let's do a followup diff after we merge both modules.

**Pull request merged**